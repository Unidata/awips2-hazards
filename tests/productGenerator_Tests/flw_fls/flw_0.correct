WGUS43 KOAX 080400
FLWOAX

TEST...FLOOD WARNING...TEST
NATIONAL WEATHER SERVICE OMAHA/VALLEY NE
1000 PM CST MON FEB  7 2011

NEC025-153-081200-
/O.NEW.KOAX.FL.W.0001.110208T0400Z-110208T1200Z/
/00000.U.ER.110208T0401Z.110208T0759Z.110208T1159Z.NO/
CASS-SARPY-
INCLUDING THE CITIES OF PLATTSMOUTH...PAPILLION...BELLEVUE AND LA
VISTA
1000 PM CST MON FEB  7 2011

...FLOOD WARNING IN EFFECT FROM 10 PM MONDAY TO 6 AM CST
TUESDAY...

THE NATIONAL WEATHER SERVICE IN OMAHA/VALLEY HAS ISSUED A

* TEST FLOOD WARNING FOR PORTIONS OF EAST CENTRAL NEBRASKA AND 
  SOUTHEAST NEBRASKA...INCLUDING THE FOLLOWING COUNTIES...IN EAST
  CENTRAL NEBRASKA...SARPY. IN SOUTHEAST NEBRASKA...CASS. 

* UNTIL 6 AM CST TUESDAY

* AT 400 AM GMT FLOODING FROM HEAVY RAIN. THIS RAIN WAS LOCATED 
  OVER THE WARNED AREA. 

* |* (OPTIONAL) POTENTIAL IMPACTS OF FLOODING *|

PRECAUTIONARY/PREPAREDNESS ACTIONS...

STAY TUNED TO FURTHER DEVELOPMENTS BY LISTENING TO YOUR LOCAL
RADIO...  TELEVISION... OR NOAA WEATHER RADIO FOR FURTHER
INFORMATION.

&&

LAT...LON 4112 9594 4100 9587 4109 9599 4112 9594




$$

