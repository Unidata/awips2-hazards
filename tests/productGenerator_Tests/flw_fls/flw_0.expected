WGUS43 KOAX 080400
FLWOAX

TEST...FLOOD WARNING...TEST
NATIONAL WEATHER SERVICE OMAHA/VALLEY NE
1000 PM CST MON FEB 7 2011



NEC025-153-081200-
/O.NEW.KOAX.FL.W.0001.110208T0400Z-110208T1200Z/
/00000.U.ER.110208T0401Z.110208T0759Z.110208T1159Z.NO/
CASS-SARPY-
INCLUDING THE CITIES OF PLATTSMOUTH...PAPILLION...BELLEVUE AND LA
VISTA
1000 PM CST MON FEB 7 2011

...FLOOD WARNING IN EFFECT FROM 10 PM MONDAY TO 6 AM CST
TUESDAY...

THE NATIONAL WEATHER SERVICE IN OMAHA/VALLEY HAS ISSUED A

* TEST FLOOD WARNING FOR EAST CENTRAL SARPY COUNTY IN EAST
CENTRAL 
  NEBRASKA NORTHEASTERN CASS COUNTY IN SOUTHEAST NEBRASKA .

* UNTIL 6 AM CST TUESDAY

* AT 1000 PM CST FLOODING FROM HEAVY RAIN. THIS RAIN WAS LOCATED
OVER THE WARNED AREA.

* |* (OPTIONAL) POTENTIAL IMPACTS OF FLOODING *|

PRECAUTIONARY/PREPAREDNESS ACTIONS...

STAY TUNED TO FURTHER DEVELOPMENTS BY LISTENING TO YOUR LOCAL
RADIO...  TELEVISION... OR NOAA WEATHER RADIO FOR FURTHER
INFORMATION.

&&

LAT...LON 4112 9594 4100 9587 4109 9599 4112 9594




$$



